test/scale-infra/deploy_ks_core.yaml

```diff
@@ -0,0 +1,124 @@
+             export INSTALL_DIR="$PWD/ocm"
+             curl -L https://raw.githubusercontent.com/open-cluster-management-io/clusteradm/main/install.sh | bash
+             export PATH=$PWD/ocm:$PATH
+        
+    - name: install KubeFlex
+      shell: |
+             sudo su <<EOF
+             bash <(curl -s https://raw.githubusercontent.com/kubestellar/kubeflex/main/scripts/install-kubeflex.sh) --ensure-folder /usr/local/bin --strip-bin
+             EOF
+
+    - name: install Helm
+      shell: |
+             curl https://baltocdn.com/helm/signing.asc | gpg --dearmor | sudo tee /usr/share/keyrings/helm.gpg > /dev/null
+             sudo apt-get install apt-transport-https --yes
+             echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/helm.gpg] https://baltocdn.com/helm/stable/debian/ all main" | sudo tee /etc/apt/sources.list.d/helm-stable-debian.list
+             sudo apt-get update
+             sudo apt-get install helm
+
+    - name: clone the clusterloader2 repo
+      shell: |
+             rm -r $HOME/perf-tests
+             git clone -b release-1.31 https://github.com/kubernetes/perf-tests.git
+
+    - name: Register a variable, ignore errors and continue
+      ansible.builtin.command: kubectl config use-context kscore
+      register: result
+      ignore_errors: true
+
+    - name: Delete old KS deployment
+      shell: |
+             kubectl config rename-context kubernetes-admin@kubernetes kscore --ignore-not-found
+             helm delete ks-core  --ignore-not-found
+             helm -n open-cluster-management delete status-addon  --ignore-not-found
+             kubectl --context kscore delete ns open-cluster-management-hub open-cluster-management  --ignore-not-found
+             kubectl config unset contexts.wds1 --ignore-not-found
+             kubectl --context kscore delete -f https://raw.githubusercontent.com/kubestellar/kubestellar/refs/heads/release-"{{ ks_release }}"/example/kind-nginx-ingress-with-SSL-passthrough.yaml  --ignore-not-found
+      when: result is success
+
+    - name: install KubeStellar using core helm-cart
+      shell: |
+             export KUBESTELLAR_VERSION="{{ ks_release }}"
+             helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart \
+             --version $KUBESTELLAR_VERSION \
+             --set-json='ITSes=[{"name":"its1","type":"host"}]' \
+             --set-json='WDSes=[{"name":"wds1"}]'
+
+    - name: Get kubeconfig "contexts" for wds1 and its1 spaces
+      shell: |
+             kflex ctx wds1
+
+    - name: install ingress controller
+      shell: |
+             kubectl config rename-context kubernetes-admin@kubernetes kscore
+             kubectl --context kscore label node --all ingress-ready="true"
+             kubectl --context kscore apply -f https://raw.githubusercontent.com/kubestellar/kubestellar/refs/heads/release-0.24.0/example/kind-nginx-ingress-with-SSL-passthrough.yaml
```

Comment:
(A) The latest regular KubeStellar release is 0.25.1.
(B) Isn't this specific to `kind`?
(C) Why not use one of the technique on https://kubernetes.github.io/ingress-nginx/deploy/ ? I also note special mention there of AWS.

Reply:
(A) I updated the ingress URL accordingly to reflect KS 0.25.1.
(B) No, it also works in a plain K8s cluster. 
(C) I am deploying my own K8s cluster using kubeadm instead of using the EKS. Hence, those technique specific for AWS don't apply in our context here.